assets/BR.yaml

```diff
@@ -0,0 +1,22 @@
+---
+title:  Baseline Requirements for the Issuance and Management of Publicly-Trusted Certificates
+subtitle:  Version 1.6.8
```

Comment:
Looks like we'll need to fix this; I can submit a PR to correct it along with the Makefile for building.

Reply:
That made me realize that I needed to update the version to 1.7.0 and add the to the version table, which I have done.